modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/AccountGroupResourceTest.java

```diff
@@ -241,6 +282,101 @@ private AccountGroup _postAccountGroup(AccountGroup accountGroup)
 		return accountGroupResource.postAccountGroup(accountGroup);
 	}
 
+	private void _testPatchAccountGroupByExternalReferenceCodeWithoutName()
+		throws Exception {
+
+		AccountGroup postAccountGroup = _postAccountGroup(randomAccountGroup());
+
+		AccountGroup randomPatchAccountGroup = randomPatchAccountGroup();
+
+		randomPatchAccountGroup.setName((String)null);
+
+		AccountGroup patchAccountGroup =
+			accountGroupResource.patchAccountGroupByExternalReferenceCode(
+				postAccountGroup.getExternalReferenceCode(),
+				randomPatchAccountGroup);
+
+		AccountGroup expectedPatchAccountGroup = postAccountGroup.clone();
+
+		BeanTestUtil.copyProperties(
+			randomPatchAccountGroup, expectedPatchAccountGroup);
+
+		expectedPatchAccountGroup.setName(postAccountGroup.getName());
+
+		AccountGroup getAccountGroup = accountGroupResource.getAccountGroup(
+			patchAccountGroup.getId());
+
+		assertEquals(expectedPatchAccountGroup, getAccountGroup);
+		assertValid(getAccountGroup);
+	}
+
+	private void _testPatchAccountGroupWithoutName() throws Exception {
+		AccountGroup postAccountGroup = _postAccountGroup(randomAccountGroup());
+
+		AccountGroup randomPatchAccountGroup = randomPatchAccountGroup();
+
+		randomPatchAccountGroup.setName((String)null);
+
+		AccountGroup patchAccountGroup = accountGroupResource.patchAccountGroup(
+			postAccountGroup.getId(), randomPatchAccountGroup);
+
+		AccountGroup expectedPatchAccountGroup = postAccountGroup.clone();
+
+		BeanTestUtil.copyProperties(
+			randomPatchAccountGroup, expectedPatchAccountGroup);
+
+		expectedPatchAccountGroup.setName(postAccountGroup.getName());
+
+		AccountGroup getAccountGroup = accountGroupResource.getAccountGroup(
+			patchAccountGroup.getId());
+
+		assertEquals(expectedPatchAccountGroup, getAccountGroup);
+		assertValid(getAccountGroup);
+	}
+
+	private void _testPutAccountGroupByExternalReferenceWithoutName()
+		throws Exception {
+
+		AccountGroup postAccountGroup = _postAccountGroup(randomAccountGroup());
+
+		AccountGroup randomAccountGroup = randomAccountGroup();
+
+		randomAccountGroup.setName((String)null);
+
+		try {
+			accountGroupResource.putAccountGroupByExternalReferenceCode(
+				postAccountGroup.getExternalReferenceCode(),
+				randomAccountGroup);
+			Assert.fail();
```

Comment:
add an empty line before

Reply:
Before the Assert.Fail or before line 347?